docs/modules/ref-tisc-workflow-components.adoc

```diff
@@ -11,8 +11,8 @@ The following table lists chosen tooling for the TISC MVP implementation.  A sep
 .Tooling by Workflow Implementation
 [cols="a,a,a",options="header"]
 |===
-| Purpose
-2+| Tools by Workflow Type
+| Component Type
+2+| Implementing Tools by Workflow Type
 
 |
 h| *Role/Module*
```

Comment:
do we need the two-tiered headers on Table 5? "Role/Module" and "Stack" complicate table and seem to provide limited value